requirements.txt

```diff
@@ -102,4 +102,4 @@ webencodings==0.5.1
 Werkzeug==1.0.1
 widgetsnbextension==3.5.1
 wrapt==1.12.1
-zipp==3.1.0
+zipp==3.1.0
```

Comment:
Where do you use this change?

Reply:
I don't think there is a change here, I think this got into this file by adding and afterwards removing a dependency.